src/components/Identification/index.js

```diff
@@ -696,7 +778,7 @@ export const Identification = props => {
                         </Typography>
                       )}
                     </div>
-                    {props.network === 'online' && (
+                    {props.network === 'online' && props.myProf && (
                       <div
```

Comment:
Due to both the My Profile and Public Profile component referencing the same Identification component, a check is made to see if the current page is the My Profile view